Formula/audacious.rb

```diff
@@ -65,31 +66,26 @@ class Audacious < Formula
   fails_with gcc: "5"
 
   def install
-    args = std_meson_args + %w[
+    args = %w[
       -Dgtk=false
-      -Dqt=true
+      -Dqt6=true
     ]
 
-    mkdir "build" do
-      system "meson", *args, "-Ddbus=false", ".."
-      system "ninja", "-v"
-      system "ninja", "install", "-v"
-    end
+    system "meson", "setup", "build", *std_meson_args, *args, "-Ddbus=false"
+    system "meson", "compile", "-C", "build", "--verbose"
+    system "meson", "install", "-C", "build"
 
     resource("plugins").stage do
       args += %w[
         -Dcoreaudio=false
         -Dmpris2=false
         -Dmac-media-keys=true
-        -Dcpp_std=c++14
```

Comment:
> What happens if you omit the custom cpp_std? We use gnu++17 [upstream](https://github.com/audacious-media-player/audacious-plugins/blob/audacious-plugins-4.3/meson.build#L6), I see no reason why this needs to be overridden by packagers.

@radioactiveman I added `-Dcpp_std=c++17` because of the failure in the previous CI run ([here](https://github.com/Homebrew/homebrew-core/actions/runs/4525347194/jobs/7969815059#step:7:791)), which clearly indicated that C++17 was not used. It must be that this `-Dcpp_std=c++14` overrode the project settings.

But since this is the default project settings, I've dropped the additional `-Dcpp_std=c++17` that I previously added.

> P.S. I have added a note for macOS on our [download page](https://audacious-media-player.org/download).

Thanks for doing that!